### PR TITLE
Add round extensions for View

### PR DESCRIPTION
**Pull request by Tougee**

### CODE REVIEW

File: src/main/java/androidx/core/view/View.kt [hunk 1]
```diff
@@ -286,3 +288,26 @@ inline fun <reified T : ViewGroup.LayoutParams> View.updateLayoutParams(block: T
...
+    clipToOutline = true
+}
+
+/**
+ * Sets the Outline to the view defined by the corner radius.
+ *
+ * @see Outline.setRoundRect
+ */
+@RequiresApi(21)
+inline fun View.round(@Px radius: Int) = round(radius.toFloat())
```
Comment (jaredsburrows) [inline src/main/java/androidx/core/view/View.kt]: 👀

Comment (romainguy) [reply src/main/java/androidx/core/view/View.kt]: I don't see why this method is needed.

File: src/main/java/androidx/core/view/View.kt [hunk 2]
```diff
@@ -286,3 +288,26 @@ inline fun <reified T : ViewGroup.LayoutParams> View.updateLayoutParams(block: T
...
     layoutParams = params
 }
+
+/**
+ * Sets the Outline to the view defined by the corner radius.
+ *
+ * @see Outline.setRoundRect
+ */
+@RequiresApi(21)
+inline  fun View.round(@Px radius: Float) {
```
Comment (romainguy) [inline src/main/java/androidx/core/view/View.kt]: This seems a little too specific. The name of the method isn't great either.

Comment (Tougee) [reply src/main/java/androidx/core/view/View.kt]: Ok I'll close it

**Comment by LOH9999** [comment]
Ok

**Pull request closed without merging**